internal/utils/utils.go

```diff
@@ -31,3 +32,9 @@ func GetSliceDiff(subset, superset []string) []string {
 	}
 	return diffs
 }
+
+func ExistsInSlice(needle string, haystack []string) bool {
```

Comment:
Does this do more than [`slices.Contains`](https://pkg.go.dev/golang.org/x/exp/slices#Contains)?